src/app/convertion-history/convertion-history.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="history-placeholder">
+    <span *ngFor="let message of historyMessages">{{message}}</span>
```

Comment:
If we had a "richer" data type, we could change the message format in one place and apply various pipes, e.g. currency.
```ts 
 <span *ngFor="let message of historyMessages">
    {{message.amountFrom | currency:message.currencyFrom:'code' }} 
    converted to 
    {{message.amountTo | currency:message.currencyTo:'code' }} 
</span>
```